### DISCUSSION TITLE
system path settings

### DISCUSSION THREAD

**Discussion by MainManPL** [otya128/winevdm#1399] [Q&A] [answered]
Hi, I'm trying to run a very old HP Chemstation from HP Kayak from 1998 I think, but the problem it requires a .dll (quite a lot of them) and it does not seem to take it from the system path set in host system (win 10 64 bit). Is there a way to define PATH for otvdm ? It also required a win.ini - but after copying one to WINDOWS folder in otvdm folder I get past that. Thanks in advance!

**Comment by emendelson** [comment]
Perhaps try putting all the .dlls into the same folder with the executable? That often works.

**Reply by MainManPL** [reply]
Thats crude :D but it worked (note that on native win7 32 bit it run properly with path settings corrected). I had to copy over 80 .dlls, so setting path still would be preferable. 
It throws some more errors, but opens window aaaaand crashes with something like that : 
![obraz](https://github.com/otya128/winevdm/assets/168545820/2eaf21b2-eea5-419f-b607-89e60b90953a).
But I think it should be different topic ...
Thanks for help.

**Comment by emendelson** [comment]
@MainManPL - Very glad to know that the dll-in-same-folder trick worked. I think it's always worked for me. I'll be curious about what happens in the other thread about this.


@cracyc - Perhaps there could be a recommendation to copy all needed dlls into the executable folder in the exe complains that it can't find a dll?

**Comment by otya128** [comment, marked as answer]
PATH can be defined using the PATH16 environment variable.
Or, you can define PATH by creating AUTOEXEC.BAT in the C directory at the same level as the WINDOWS directory and writing `SET PATH=C:\PATH\TO\;`.

directory structure:
* otvdm.exe
* WINDOWS
    * SYSTEM
    * ...
* C
    * AUTOEXEC.BAT

**Reply by emendelson** [reply]
Ah! I should have remembered that. I'll try to add a PATH16 setting to my InnoSetup installer.

**Reply by MainManPL** [reply]
Thanks, I'll check that.
